app/controllers/collections_controller.rb

```diff
@@ -17,7 +17,7 @@ def edit
   end
 
   def show
-    @collection = Collection.find_by_druid(params[:id])
+    @collection = Collection.find(params[:id])
```

Comment:
why does `find` vs `find_by_druid` matter to this?

Reply:
This app is an API as well as a user facing CRUD app.  As the API user won't know the internal primary key, they are using druids as a proxy key.   I've set the `Collection.primary_key = 'druid'` in this PR, so we don't need the `_by_druid` part because now all finds are `_by_druid`

Reply:
thanks.  Still wonder how a user would go about setting the last crawl month (or if that would be done by a user).

Reply:
They could do it from the edit form.